lib/receptor_controller/client/directive_blocking.rb

```diff
@@ -0,0 +1,86 @@
+require "receptor_controller/client/directive"
+
+module ReceptorController
+  # Blocking directive for requests through POST /job
+  # Successful POST causes locking current thread until response from Kafka comes
+  #
+  # Raises kind of ReceptorController::Client::Error in case of problems/timeout
+  class Client::DirectiveBlocking < Client::Directive
```

Comment:
So - I'm ok with leaving this in for now, but I heavily think any implementation needs to be done with the `NonBlocking` directive. Since we are going to be moving to the external redis cache for storing the `msg_id`'s that lines up with the non blocking directive.

Reply:
I'm ok with that but for first version it's easier to use blocking directive for subqueries (surveys, credentials etc.). The first PR is big enough not to make all changes in one shot